fulp_modules/features/antagonists/bloodsuckers/code/bloodsucker/bloodsucker_life.dm

```diff
@@ -420,7 +420,7 @@
 	timeout = 10 MINUTES
 
 /datum/mood_event/madevamp
-	description = "<span class='boldwarning'>A soul has been cursed to undeath by my own hand.</span>\n"
+	description = "<span class='boldwarning'>A soul has reached an apotheosis- undeath- by my own hand.</span>\n"
```

Comment:
This still isn't correct as per the VTM/WOD lore I can find. The soul of a vampire is still living, they are simply inhabiting dead bodies. Embraced mortals get a portion of their sire's soul, so perhaps it could be reframed to include that, instead

Reply:
I wrote it this way to make it be more positive. “I cursed a living soul with undeath” just doesn’t sound like the description of a positive moodlet, because the negative connotation of “cursed” implies some sort of regret. This was a rewrite of the existing line, which could still be correct under lore, as a “soul” may refer to the _person_ rather than the soul itself.

Reply:
> as a “soul” may refer to the person rather than the soul itself.

No, souls exist in World of Darkness pretty definitively. It'd be weird to interpret it as the uncommon synonym rather than its literal definition

If you want to make it more positive, just frame it around sharing your soul with the other vampire, as that is what makes the connection between embraced and sires so strong

Reply:
https://whitewolf.fandom.com/wiki/Soul_(WOD) 

For reference, "World of Darkness" is the name of the overall setting VTM takes place in